examples/rvc-app/run_all_yaml_tests.sh

```diff
@@ -0,0 +1,40 @@
+#!/bin/bash
+
+## ---
+# This is a helper script that runs all the yaml tests for all the application clusters enabled in the RVC App.
+# Run the script from the root dir.
+# The script takes the node ID that the device was commissioned with.
+## ---
+
+NODEID=$1
+RVC_DEVICE_ENDPOINT=1
+
+PICS_RUN="examples/rvc-app/rvc-common/pics/RVC_Clean_Mode_Cluster_Test_Plan.txt"
+PICS_CLEAN="examples/rvc-app/rvc-common/pics/RVC_Run_Mode_Cluster_Test_Plan.txt"
+PICS_OP_STATE="examples/rvc-app/rvc-common/pics/RVC_Operational_State_Cluster_Test_Plan.txt"
+
+if [ -z "$NODEID" ]; then
+    echo "Usage: run_all_yaml_tests [Node ID]"
+    exit
+fi
+
+# RVC Clean Mode cluster
+./scripts/tests/yaml/chiptool.py tests Test_TC_RVCCLEANM_1_1 --PICS "$PICS_CLEAN" --nodeId "$NODEID" --endpoint "$RVC_DEVICE_ENDPOINT" &&
+    ./scripts/tests/yaml/chiptool.py tests Test_TC_RVCCLEANM_1_2 --PICS "$PICS_CLEAN" --nodeId "$NODEID" --endpoint "$RVC_DEVICE_ENDPOINT" &&
+    ./scripts/tests/yaml/chiptool.py tests Test_TC_RVCCLEANM_2_1 --PICS "$PICS_CLEAN" --nodeId "$NODEID" --endpoint "$RVC_DEVICE_ENDPOINT" &&
+    ./scripts/tests/yaml/chiptool.py tests Test_TC_RVCCLEANM_3_1 --PICS "$PICS_CLEAN" --nodeId "$NODEID" --endpoint "$RVC_DEVICE_ENDPOINT" &&
```

Comment:
TC 3.1 and 3.3 don't apply to the RVC device types since the DEPONOFF feature is false. 
TC 3.2 is a Python test so it should be removed from this list. 

This comment also applies for the RVC Run Mode tests, below.

Reply:
Is there a reason why the TC 3.2 exists as a yaml if there is a python test?

I have update these [here](https://github.com/project-chip/connectedhomeip/pull/29159/commits/83cd02de4a2c7121fde44e0127f4490f2faf5c79).

Reply:
The RVC Run and Clean TC 3.2 yaml files were removed. Do you still see them somewhere?

Reply:
@mhazley William removed TCs 3.1 and 3.3 (see [this](https://github.com/project-chip/connectedhomeip/pull/29159/commits/83cd02de4a2c7121fde44e0127f4490f2faf5c79)). I'm not sure why you added them back.

Reply:
Sorry - I ran the tests yesterday and 1_2 and 2_1 didn't seem to exist - when I looked at the repo I saw 3_1 and 3_3 so assumed it was a typo

Reply:
@plauric It looks like they where removed after I had written this script. There where also changes to the RVC mode clusters yamls since then. That in what @mhazley's commit was addressing but I think I forgot to mention to remove those tests.

Reply:
Fixed [here](https://github.com/project-chip/connectedhomeip/pull/29159/commits/234824b1361efb34de8a5f6763bfc44b3a80590d).